src/other/Table.js

```diff
@@ -529,27 +567,35 @@
                     }
                 }            
             }
-            context.tableDiv
-                .style("width", finalWidth)
-                .style("height", newTableHeight)
-            ;
+            if (element.classed("childWidget")) {
+                context._parentElement
+                    .style("width", finalWidth)
+                    .style("height", newTableHeight)
+                ;
+                context.tableDiv
+                    .style("overflow", "hidden")
+                ;
+            }
+            context.size({width:parseInt(finalWidth), height: parseInt(newTableHeight)});
         }
 
-        this.setOnScrollEvents(this.tableDiv.node(), tableMarginHeight);
+        if (element.classed("childWidget")) {
+            this._parentWidget.setColumnWidths();
+        }
 
+        this.setOnScrollEvents(this.tableDiv.node(), tableMarginHeight);
 
         function calcWidth() {
-            var newTableWidth = box.width - parseInt(fixedColWidth);
-            maxWidth = context.table.node().offsetWidth - fixedColWidth + context.getScrollbarWidth();
+            var newTableWidth = box.width;
+            maxWidth = context.table.node().offsetWidth + 1;
             finalWidth = newTableWidth > maxWidth ? maxWidth : newTableWidth;
             finalWidth = finalWidth + "px";
         }
 
         function calcHeight() {
-            newTableHeight = box.height - tableMarginHeight + context.getScrollbarWidth();
+            newTableHeight = context.tbody.node().offsetHeight + parseInt(tableMarginHeight) + 1;
```

Comment:
Can you access "offsetrHeight" via context.tbody.attr("offsetHeight") or context.tbody.property("offsetHeight")?  If so it would be "safer".